backend/data/schema/schema.go

```diff
@@ -11,6 +11,7 @@ type User struct {
 	FirebaseID string
 	Username   string
 	Email      string
+	Events		[]string // FirebaseID of events that this user is attending or the owner of
```

Comment:
This is just a tiny change I snuck in here. I can make a seperate PR just lmk.

Reply:
nvm github is struggling and the diff is incorrect - this change is already merge in. The change in this file is a removed comment that I added

Reply:
Might have to merge in the latest master for this diff to show up correctly - not a huge deal though.